app/src/ui/changes/changed-file.tsx

```diff
@@ -9,6 +9,8 @@ import { Checkbox, CheckboxValue } from '../lib/checkbox'
 
 const GitIgnoreFileName = '.gitignore'
 
+const RestrictedFileExtensions = ['.cmd', '.exe', '.bat', '.sh']
```

Comment:
Yeah, I don't intend for this to be a permanent solution. I'm just worried about the "executing arbitrary code" scenario and distrusting repositository content.